api/reference/authentications.adoc

```diff
@@ -0,0 +1,202 @@
+
+[[authentications]]
+== Authentications
+
+Management of Authentications is provided via the following collection:
+
+[source,data]
+----
+/api/authentications
+----
+
+Full CRUD actions on authentications is available:
+
+* link:#querying-authentications[Querying Authentications]
+* link:#creating-authentications[Creating Authentications]
+* link:#editing-authentications[Editing Authentications]
+* link:#deleting-authentications[Deleting Authentications]
+
+[[querying-authentications]]
+=== Querying Authentications
+
+Querying all authentications in the system is simply:
+
+----
+GET /api/authentications
+----
+
+Getting details on a specific Authentication:
```

Comment:
Any reason for capitalizing "Authentication" here (and not above)? Consistent would be better, and personally prefer lowercase